_posts/2021-01-10-boa-release-11.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Boa release v0.11"
+author: Boa Developers
+---
+
+Boa has reached a new release. v0.11, our biggest one yet. Since v0.10 we've closed 77 issues, merged 129 pull requests. The engine has been faster and more compliant to the spec. Below are some of the highlights but please see the [changelog link] for more information.
+
+## Test 262
```

Comment:
In this section we could add an image of the progress, like the one in https://github.com/boa-dev/boa/pull/1065.

Reply:
Do you have a light and dark mode version of that picture?

Reply:
We have no dark mode, but I will create the pictures and send them to you via Discord :)

Reply:
Thanks, if the background could match the color of https://boa-dev.github.io/ that would be nice